RELEASE-NOTES.txt

```diff
@@ -1,5 +1,13 @@
 Unreleased
 ---
+1.62.2
+------
+* Same as 1.62.1 but with a gutenberg changelog.
+
+1.62.1
```

Comment:
@enejb since this release won't happen I think we can just remove it altogether and put the changelog under 1.62.2. WDYT?

Reply:
This is not a blocker though, I am seeing that you simply extended the previous release changes and bumped the version to `1.6.2.2` so the `1.62.1` release is indeed included, so now looking at it, it's fine 👍🏾